src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -968,7 +969,7 @@ ucs_status_t uct_dc_mlx5_ep_pending_add(uct_ep_h tl_ep, uct_pending_req_t *r,
     uct_dc_mlx5_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_dc_mlx5_iface_t);
     uct_dc_mlx5_ep_t *ep = ucs_derived_of(tl_ep, uct_dc_mlx5_ep_t);
 
-    /* ep can tx iff
+    /* ep can tx if
```

Comment:
why the change?

Reply:
typo fix. 
remove?

Reply:
it's not a typo, iff means "if and only if"

Reply:
ok, I did not know it